ZelBack/src/services/syncthingService.js

```diff
@@ -1757,11 +1775,15 @@ async function debugSupport(req, res) {
   const authorized = res ? await verificationHelper.verifyPrivilege('adminandfluxteam', req) : true;
   let response = null;
   if (authorized === true) {
-    response = await performRequest('get', '/rest/debug/support', undefined, 60000);
+    try {
+      response = await performRequest('get', '/rest/debug/support', undefined, 60000);
```

Comment:
performRequest is has already try catch and practically can't throw

Reply:
Oh, fixed it.